app/views/step_by_step_pages/new.html.erb

```diff
@@ -1,22 +1,17 @@
 <%
   links = [
     {
-      text: 'Step by step publisher',
+      text: "Step by step publisher",
```

Comment:
This is unchanged apart from double quotes - we use single quotes in most places in the other edited files, and this file was mostly single quotes before the change too. Could you look at that improving that consistency?